web_app/hazard-reporting-map/package.json

```diff
@@ -9,10 +9,12 @@
     "lint": "next lint"
   },
   "dependencies": {
```

Comment:
you havent use these two packages . please remove it if its not necessary

Reply:
I have removed unused packages